libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -492,6 +492,9 @@ class Config:
     max_retries: int = 2
     """Number of retries allowed for requests sent to the Anthropic Completion API."""
 
+    stop: Optional[List[str]] = Field(None, alias="stop_sequences")
```

Comment:
can we make the attribute stop_sequences and the alias stop? since stop_sequences is the Anthropic name for it